lib/core/macros.nim

```diff
@@ -1076,20 +1076,24 @@ proc expectKind*(n: NimNode; k: set[NimNodeKind]) {.compileTime.} =
   ## macros that check the AST that is passed to them.
   if n.kind notin k: error("Expected one of " & $k & ", got " & $n.kind, n)
 
-proc newProc*(name = newEmptyNode(); params: openArray[NimNode] = [newEmptyNode()];
-    body: NimNode = newStmtList(), procType = nnkProcDef): NimNode {.compileTime.} =
+proc newProc*(name = newEmptyNode();
+              params: openArray[NimNode] = [newEmptyNode()];
+              body: NimNode = newStmtList();
+              procType = nnkProcDef;
+              pragma: NimNode = newEmptyNode()): NimNode {.compileTime.} =
```

Comment:
It should be ``pragmas``, plural.

Reply:
Do you mean new commit or ``pragmas: openArray[NimNode]`` ?
If you mean later, should pragma be nnkIdent?

Reply:
All I'm saying is that parameter's name shall be ``pragmas``.

Reply:
Thanks.
Then, the newer commit is fixed one.
Thanks for retriggering CI.